content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -547,6 +547,19 @@ This annotation requires the [NodePreferAvoidPods scheduling plugin](/docs/refer
 to be enabled. The plugin is deprecated since Kubernetes 1.22.
 Use [Taints and Tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/) instead.
 
+### preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority
+
+Example: `preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority: "10000"`
+
+This specifies the minimum priority value that can preempt this [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass). It defaults to the PriorityClass's priority value + 1 if not set, which means pods that have a higher priority value can preempt it.
+
+### preemption-toleration.scheduling.sigs.k8s.io/toleration-seconds
+
+Example: `preemption-toleration.scheduling.sigs.k8s.io/toleration-seconds: "3600"`
+
+This specifies how long the [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) can tolerate preemption
+by priorities lower than `MinimumPreemptablePriority`. It defaults to zero if not set. Zero value means the pod will be preempted immediately. i.e., no toleration at all. If it's set to a positive value, the duration will be honored. If it's set to a negative value, the pod can be tolerated forever - i.e., pods with priority lower than `MinimumPreemptablePriority` won't be able to preempt it. This value affects scheduled pods only (no effect on nominated pods).
```

Comment:
- Is this an annotation or a label?
- What kind of objects is this used on?
- (nit) [Avoid using Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations

Reply:
Readers may also wonder what `MinimumPreemptablePriority` is. Can we help people to work that out?

Reply:
`MinimumPreemptablePriority` is the annotation that is defined just above for PriorityClass, we can link this annotaion here but as it is defined just above so linking it doesn't making much sense to me. What do you think?

Reply:
> * (nit) [Avoid using Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations
ok, i will follow it.